### ISSUE TITLE
can not get the good result in paper

### ISSUE THREAD

**Issue by csufangyu** [shikharbahl/neural-dynamic-policies#2]
I run the code in RL,but I find there maybe some error.and I can not get the good result in paper,could you help me? My success rate always is  0.0,and I always get the error is
`mujoco_py.builder.MujocoException: Got MuJoCo Warning: Nan, Inf or huge value in QPOS at DOF 0. The simulation is unstable. Time = 2321.0500.`

**Comment by shikharbahl** [comment]
Hi,

What exact command(s) did you run (`--env`, `--type`, `--T`, `--N`, etc)? This mujoco error is usually caused by the policy producing very large actions.

**Comment by csufangyu** [comment]
hi I run the the commands is :`python ./main_rl.py --env-name "throw"  --type "dmp" --seed 0 --expID 1 --reward-delay 5 --T 5 --N 5 --a_z 5
` 
I use torch.clamp to fixed it,but I read the code and paper，I am very confused.could you help me?thanks very much

**Comment by csufangyu** [comment]
and I run the code,can not get as well as in paper,can you share the hyperparameter？ I am very interesting this work,It combines traditional robot dynamics models and deep learning，I think it maybe help us formulate a more reasonable robot learning policy. Thanks very much!

**Comment by linksdl** [comment]
Hi, @csufangyu  When I run this project, I can not found the './dmp/data/40x40-smnist.mat', Could you help me solve this problem?

**Comment by shikharbahl** [comment]
Hi @linksdl I uploaded the data folder!

**Comment by shikharbahl** [comment]
@csufangyu Could you share the exact torch and mujoc versions you are using?

**Comment by RayYoh** [comment]
> Hi @linksdl I uploaded the data folder!

So, where is the folder?
